### ISSUE TITLE
Web component: Tabs extended &  Tabs extended with media - Keyboard tab focus issues. - DDS consulting

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5816]
## Detailed description

Issue:
For keyboard only users
1.Each tabs are receiving the tab focus.
2.We are not able to navigate between the tabs with right/left arrow keys.

Expected:
The only selected tab should receive the tab focus and rest of the tabs should be navigable with right/left arrow keys.

## Additional information
-  Engineering info:  
     - HC engineer:
     - JC Jira Ticket: https://jsw.ibm.com/browse/HC-2191
     - DDS consulting engineer:   Anna Wen

**Comment by RobertaJHahn** [comment]
@Praveen-sr Is the left and right arrow support required for IBM accessibility standards?  Will this support also be required for Masthead navigation?

**Comment by Praveen-sr** [comment]
@RobertaJHahn Yes its a IBM and WCAG accessibility requirement.
The arrow key behavior only required wherever we have Tabs, Therefore its required for Tabs extended & Tabs extended but not for Masthead.
Masthead is having menu options not tabs.
Here is the link for your reference:- https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html
